csrc/predicate_compute.cpp

```diff
@@ -61,6 +61,14 @@ Bool* ParallelizedDomainPredicate::PredicateInfo::getPredicate() const {
         GpuLower::current()->caMap()->getConcreteMappedID(
             pred_id, IdMappingMode::EXACT));
     auto new_pred = SimplifyingIrBuilder::ltExpr(index, pred_id->extent());
+    // pt_ not being exact does not mean the predicate is not trivial. For
```

Comment:
This makes sense, but wouldn't it be nicer if such IDs were not added to the predicate ID list from the beginning? Can this filtering be done at around line 211?

Reply:
I agree with what you said. Let me give it a try.

Reply:
fixed, it's working